### PR TITLE
Fix for Dynamcis Integration Bug

### PR DESCRIPTION
**Pull request by javiergas**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. Excute Dynamics Integration from Campaing or Form Action
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.

### CODE REVIEW

File: plugins/MauticCrmBundle/Integration/DynamicsIntegration.php [hunk 1]
```diff
@@ -413,7 +413,7 @@ public function pushLead($lead, $config = [])
      */
     public function getLeads($params = [], $query = null, &$executed = null, $result = [], $object = 'contacts')
     {
-        if ('Contact' === $object) {
+        if ('Contacts' === $object) {
```
Comment (kuzmany) [inline plugins/MauticCrmBundle/Integration/DynamicsIntegration.php]: Need check both Contact and Contacts - see our PR https://github.com/mautic/mautic/pull/5674/files#diff-739ed4b59971b6352ae731f734796c5fR438
Both fixes are in uncomming big update https://github.com/mautic/mautic/pull/5674. @escopecz  consider duplicate label

**Comment by kuzmany** [comment]
Label: Pending Test Confirmation

**Comment by kuzmany** [comment]
Label: Ready To Test

**Comment by kuzmany** [comment]
Label: Pending Feedback

File: plugins/MauticCrmBundle/Integration/DynamicsIntegration.php (hunk 1 above)
Comment (escopecz) [reply plugins/MauticCrmBundle/Integration/DynamicsIntegration.php]: Yep, #5674 seems to consider more use cases. I'll close this one.

**Pull request closed without merging**